formula-coroutines/src/main/java/com/instacart/formula/coroutines/FlowFormula.kt

```diff
@@ -1,18 +0,0 @@
-package com.instacart.formula.coroutines
-
-import com.instacart.formula.Action
-import com.instacart.formula.ActionFormula
-import kotlinx.coroutines.flow.Flow
-
-abstract class FlowFormula<Input : Any, Output : Any> : ActionFormula<Input, Output>() {
```

Comment:
I'm not sure about this type. It doesn't provide much value over `ActionFormula`